docs/EKSManualDeploymentGuide.md

```diff
@@ -22,3 +22,9 @@ Prepare kube config file
 ```
 $ aws eks --region <REGION_NAME> update-kubeconfig --name <EKS_CLUSTER_NAME>
 ```
+
+## Next steps
+
+Once the EKS cluster is ready you can keep going by following:
```

Comment:
Ditto.

Reply:
Yes, It is redundant.
We can remove this documentation because we have a detailed document for `Manual deployment guide ScalarDL on EKS`